testsuite/tests/typing-layouts-arrays/test_ignorable_product_array_1.ml

```diff
@@ -0,0 +1,93 @@
+(* TEST
```

Comment:
@rtjoa @ccasin These `test_*_product_array_*.ml` tests don't seem to be checking the locally-allocating versions of the primitives.  However the new autogenerated tests do, so maybe this is ok?

Reply:
I think it's OK for the reason you identified.  I believe @rtjoa looked at this and judged it easier to write those new tests than adapt this framework.